recipes/parro/README.md

```diff
@@ -0,0 +1,7 @@
+# Parro for Ferdium
```

Comment:
We don't really need this file. Please remove it.

Reply:
done